index.js

```diff
@@ -27,6 +27,9 @@ class ServerlessSecretBaker {
 
   async writeEnvironmentSecretToFile() {
     const providerSecrets = this.serverless.service.provider.environmentSecrets;
+    if (!providerSecrets) {
```

Comment:
did you test this? I'm curious because it looks like, even with this change, it will still attempt to include 'secretsFile' in the package.  However, with this change, we don't write the file, and I wonder if the included a missing file will bother serverless.

Reply:
Yes, no file showed up in the zip package

No secret specified:
![Screen Shot 2019-08-19 at 4 55 55 PM](https://user-images.githubusercontent.com/1832296/63307241-9877f680-c2a2-11e9-817b-4ce0d0fc0284.png)


Secret specified:

![Screen Shot 2019-08-19 at 4 57 54 PM](https://user-images.githubusercontent.com/1832296/63307248-9ca41400-c2a2-11e9-876d-fd85c05124d5.png)